locale/action_names.rb

```diff
@@ -0,0 +1,7 @@
+# Autogenerated!
+_("Action with sub plans")
+_("Create architecture")
+_("Delete architecture")
+_("Update architecture")
+_("Import facts")
+_("")
```

Comment:
@iNecas Any idea how to get it to not try the interpolation when it extracts the action names?